dissect/target/loaders/dir.py

```diff
@@ -15,11 +15,20 @@
 class DirLoader(Loader):
     """Load a directory as a filesystem."""
 
+    PREFIXES = ["", "fs"]
+
+    @classmethod
+    def _find_entry_path(cls, path: Path) -> str | None:
```

Comment:
Maybe you can make this a function in the global scope, similar to the other functions here. There's not much reason for it to be a classmethod, is there?